src/EnergyPlus/ZoneEquipmentManager.cc

```diff
@@ -2609,10 +2610,12 @@ namespace ZoneEquipmentManager {
 				z.OutHumRatAtHeatPeak = c.OutHumRatAtHeatPeak;
 				z.TimeStepNumAtHeatMax = c.TimeStepNumAtHeatMax;
 				z.DesHeatVolFlow = c.DesHeatVolFlow;
+				z.NonAirSysDesHeatVolFlow = c.DesHeatVolFlow;
 				z.DesHeatCoilInTemp = c.DesHeatCoilInTemp;
 				z.DesHeatCoilInHumRat = c.DesHeatCoilInHumRat;
 
 				z.DesCoolLoad = c.DesCoolLoad;
+				z.NonAirSysDesCoolLoad = c.DesCoolLoad;
```

Comment:
Correct, I didn't add the CoolVol variable because it would not be used anywhere. I'll circle back when I see what the CI reports to fix the description. I'm on the fence about adding in the unused variable.